lib/localization/locales/tr.json

```diff
@@ -0,0 +1,63 @@
+  "hack_purpose": "Bu geçici bir çözümdür. Esas amacı, başsız tarayıcıları parmak iziyle tanımlamak için daha fazla zaman kazandırmak, ve bu sayede meşru kullanıcıların bu zorluk sayfasını görmesini önlemektir.",
+  "jshelter_note": "Lütfen dikkat: Anubis, JShelter gibi eklentilerin devre dışı bıraktığı modern JavaScript özelliklerini gerektirir. Lütfen bu alan adı için JShelter veya benzeri eklentileri devre dışı bırakın.",
+  "version_info": "Bu web sitesi şu Anubis sürümünü çalıştırıyor:",
+  "try_again": "Tekrar dene",
+  "go_home": "Ana sayfaya dön",
+  "contact_webmaster": "ya da engellenmemeniz gerektiğini düşünüyorsanız, lütfen şu adrese e-posta gönderin:",
+  "connection_security": "Bağlantınızın güvenliğini sağlarken lütfen bekleyin.",
+  "javascript_required": "Ne yazık ki bu aşamayı geçebilmek için JavaScript’i etkinleştirmeniz gerekiyor. Bunun nedeni, AI şirketlerinin web barındırma konusundaki sosyal sözleşmeyi değiştirmiş olmasıdır. JavaScript’siz bir çözüm geliştirilmektedir.",
+  "benchmark_requires_js": "Kıyaslama aracının çalıştırılması için JavaScript’in etkin olması gereklidir.",
+  "difficulty": "Zorluk:",
+  "algorithm": "Algoritma:",
+  "compare": "Karşılaştır:",
+  "time": "Süre",
+  "iters": "Tekrar",
+  "time_a": "Süre A",
+  "iters_a": "Tekrar A",
+  "time_b": "Süre B",
+  "iters_b": "Tekrar B",
+  "static_check_endpoint": "Bu sadece ters proxy'nizin kullanması için bir kontrol adresidir.",
+  "authorization_required": "Yetkilendirme gerekli",
+  "cookies_disabled": "Tarayıcınız çerezleri devre dışı bırakacak şekilde yapılandırılmış. Anubis, geçerli bir istemci olduğunuzu doğrulamak için çerezlere ihtiyaç duyar. Lütfen bu alan adı için çerezleri etkinleştirin.",
+  "access_denied": "Erişim Reddedildi: hata kodu",
+  "dronebl_entry": "DroneBL bir giriş bildirdi",
+  "see_dronebl_lookup": "bakınız",
+  "internal_server_error": "Dahili Sunucu Hatası: Yönetici Anubis'i yanlış yapılandırmış. Lütfen yöneticinizle iletişime geçin ve şunun civarındaki kayıtlara bakmasını isteyin:",
+  "invalid_redirect": "Geçersiz yönlendirme",
+  "redirect_not_parseable": "Yönlendirme URL'si çözümlenemiyor",
+  "redirect_domain_not_allowed": "Yönlendirme alan adına izin verilmiyor",
+  "failed_to_sign_jwt": "JWT imzalanamadı",
+  "invalid_invocation": "Geçersiz MakeChallenge çağrısı",
+  "client_error_browser": "İstemci Hatası: Lütfen tarayıcınızın güncel olduğundan emin olun ve daha sonra tekrar deneyin.",
+  "oh_noes": "Ah hayır!",
+  "benchmarking_anubis": "Anubis kıyaslanıyor!",
+  "you_are_not_a_bot": "Bot değilsiniz!",
+  "making_sure_not_bot": "Bot olmadığınızdan emin oluyoruz!",
+  "celphase": "CELPHASE",
+  "js_web_crypto_error": "Tarayıcınızda çalışan bir web.crypto öğesi yok. Bu sayfayı güvenli bir bağlantı üzerinden mi görüntülüyorsunuz?",
+  "js_web_workers_error": "Tarayıcınız web worker'ları desteklemiyor (Anubis, tarayıcınızın donmaması için bunları kullanır). JShelter gibi bir eklenti mi kurulu?",
```

Comment:
maybe "web iscileri" not really sure though

Reply:
Yeah, that makes sense. Neither would be familiar to the average user anyway.

Reply:
Fixed in 65adf59.

Reply:
Now that I think about it web workers is just a weird name anyways. When I think of "web iscileri" I'm reminded of amazon mechanical turk or similar, but the english one lacks the same connotation to me.

Perhaps "web iscileri adli tarayici ozelligini desteklemiyor" or similar?